python/fi_instrumentation/otel.py

```
@@ -418,19 +446,194 @@ class BatchSpanProcessor(_BatchSpanProcessor):
+        sig = _get_class_signature(_GRPCSpanExporter)
+        bound_args = sig.bind_partial(*args, **kwargs)
+        bound_args.apply_defaults()
+
+        auth_header = get_env_fi_auth_header()
+        lower_case_auth_header = (
+            {k.lower(): v for k, v in auth_header.items()} if auth_header else None
+        )
+
+        if not bound_args.arguments.get("headers"):
+            headers = {
+                **(lower_case_auth_header or dict()),
```

Pylint (3.3.7)

[refactor] Consider using '{}' instead of a call to 'dict'.

(R1735)